manifests/config.pp

```diff
@@ -83,7 +89,7 @@
       comment => 'Foreman',
       home    => $foreman::app_root,
       gid     => $foreman::group,
-      groups  => $foreman::user_groups,
+      groups  => $_user_groups,
```

Comment:
I think this works, but there's a risk someone added `puppet` already and it ends up being added twice. We may also need an installer migration for this.
```suggestion
      groups  => unique($_user_groups),
```